.gitignore

```diff
@@ -0,0 +1,17 @@
+# GoLand IDE files (project definition, etc)
+.idea
+
+# Binaries for programs and plugins
+*.exe
+*.dll
+*.so
+*.dylib
+
+# Test binary, build with `go test -c`
+*.test
+
+# Output of the go coverage tool, specifically when used with LiteIDE
+*.out
+
+# Project-local glide cache, RE: https://github.com/Masterminds/glide/issues/736
+.glide/
```

Comment:
I thought we weren't going to use any package manager...

Reply:
Better do in a separate PR if you want to add a `.gitignore` file. I actually kind of like the freedom to have my own `.gitignore` 🙊